command/service_info.go

```diff
@@ -173,6 +173,13 @@ func (s *ServiceInfoCommand) formatOutput(jobIDs []string, jobServices map[strin
 	s.Ui.Output(formatList(outputTable))
 }
 
+func formatAddress(address string, port int) string {
+	if port == 0 {
+		return address
+	}
+	return fmt.Sprintf("%s:%d", address, port)
```

Comment:
I'm about to ask the question no developer wants to hear: what about ipv6?

https://pkg.go.dev/net#JoinHostPort

```suggestion
	return net.JoinHostPort(address, strconv.Itoa(port))
```